docs/sphinx/source/installation.rst

```diff
@@ -34,49 +33,153 @@ There are many ways to install Python on your system, but the Anaconda
 Scientific Python distribution provides by far the easiest way for new
 users to get started. Anaconda includes all of the popular libraries
 that you'll need for pvlib, including Pandas, NumPy, and SciPy.
-"Anaconda installs cleanly into a single directory, does not require
-Administrator or root privileges, does not affect other Python installs
-on your system, or interfere with OSX Frameworks." -Anaconda
-Documentation.
 
-#. Install the full Anaconda Scientific Python distribution available
-   `here <https://store.continuum.io/cshop/anaconda/>`_
-#. Install pvlib: ``conda install -c pvlib pvlib``
+    Anaconda installs cleanly into a single directory, does not require
+    Administrator or root privileges, does not affect other Python installs
+    on your system, or interfere with OSX Frameworks. -- `The Anaconda
+    Documentation <https://docs.continuum.io/anaconda/index>`_
+
+#. **Install** the full Anaconda Scientific Python distribution available
+   `at Continuum.io <https://store.continuum.io/cshop/anaconda/>`_
+
+See the `Anaconda FAQ <http://docs.continuum.io/anaconda/faq.html>`_
+for more information.
+
+
+.. _standardrelease:
+
+Install standard release
+------------------------
+
+To obtain the most recent stable release of pvlib-python, use
+`conda <http://conda.pydata.org/docs/>`_ or `pip <https://pip.pypa.io>`_::
+
+    conda install -c pvlib pvlib
+
+    pip install pvlib
+
```

Comment:
I suggest:

conda install -c pvlib pvlib
# or, if not using conda, e.g. on WinPython:
pip install pvlib